lago/prefix.py

```diff
@@ -645,7 +645,10 @@ def _handle_lago_template(
             template_store.get_stored_metadata(template_version, ),
         )
         base = template_store.get_path(template_version)
-        qemu_cmd = ['qemu-img', 'create', '-f', 'qcow2', '-b', base, disk_path]
+        qemu_cmd = [
+            'qemu-img', 'create', '-f', 'qcow2', '-o', 'compat=1.1', '-o',
+            'lazy_refcounts=on', '-b', base, disk_path
+        ]
```

Comment:
Do we also need to set the device cache definition to 'writethrough'? Couldn't figure if that is the default. I don't think we add any definition now(by dumping the xml from libvirt).
```
   "lazy_refcounts"
               If this option is set to "on", reference count updates are postponed with the goal of avoiding metadata I/O and improving performance. This is particularly interesting with
               cache=writethrough which doesn't batch metadata updates. The tradeoff is that after a host crash, the reference count tables must be rebuilt, i.e. on the next open an (automatic) "qemu-img
               check -r all" is required, which may take some time.
```
qemu-kvm:
```
By default, the cache=writeback mode is used. It will report data writes as completed as soon as the data is present in the host page cache.  This is safe as long as your guest OS makes sure to
           correctly flush disk caches where needed. If your guest OS does not handle volatile disk write caches correctly and your host crashes or loses power, then the guest may experience data
           corruption.
```